CHANGES.rst

```diff
@@ -5,12 +5,14 @@ Version 0.9.0 (UNRELEASED)
 ---------------------------
 
 - Adds support for Kubernetes networking/v1 API to interactive sessions.
+- Adds setup of Kerberos when executing a workflow engine.
+- Adds the `REANA_WORKSPACE` environment variable to jupyter notebooks and terminals.
+- Changes workflow create endpoint to populate workspace retention rules for the workflow.
```

Comment:
I think we should move this line to the first commit, given that it is not related to this commit